docs/game-mechanics/behaviors/aoe.rst

```diff
@@ -1,36 +1,59 @@
 Area of Effect / AoE (7)
 ========================
 
-This behavior calls the specified action on all / a maximum of players
-around it.
+This behavior calls the specified action on all / a maximum number of entities in the casters' radius.
 
 Parameters
 ----------
 
 .. list-table ::
-   :widths: 15
+   :widths: 15 30
    :header-rows: 1
 
    * - Name
+     - Description
    * - action
+     - The behavior to be performed.
    * - ignore_faction
+     - A faction to ignore during targetting.
    * - ignore_faction1
+     - A faction to ignore during targetting.
    * - ignore_faction2
+     - A faction to ignore during targetting.
    * - ignore_faction3
+     - A faction to ignore during targetting.
    * - include_faction
+     - A faction to include in targetting
    * - max targets
+     - The maximum number of allowed targets.
    * - radius
+     - The radius to check for targets in.
    * - target_enemy
+     - Whether or not to target an enemy.
    * - target_friend
+     - Whether or not to target a friend.
    * - target_self
+     - Whether or not to target self.
    * - target_team
+     - Whether or not to target a team.
    * - use_target_as_caster
+     - Whether or not to use the target as the caster.
    * - use_target_position
+     - Whether or not to use the targets' position.
 
 BitStream Serialization
 -----------------------
 
-| **[u32]** - number of targets
-| 	**[s64]** - target object id
+| **[u32]** - The number of targets.
+| **[s64]** - The target object id.
 | **[for target in targets]**
-|   -> action(target)
+| -> action(target)
+
+Darkflame Universe BitStream Serialization Notes
+------------------------------------------------
+
+When reading the BitStream in, the number of target object ids should be 
+less than or equal to the number of targets.
```

Comment:
Can you elaborate on the less than case? That shouldn't happen afaik

Reply:
The only reason I have the less than case is because in dlu source, we only check for if `if (targetCount > this->m_maxTargets)` and not `if (targetCount == this->m_maxTargets)`.  I too think they believe they need to be equal but thats just my understanding from dlu source

Reply:
That's just a sanity check though afaict. The actual number is used for the loop is whatever is read from the packet

https://github.com/DarkflameUniverse/DarkflameServer/blob/42f6f2f10b5971dd13faa18e2018892ce21ce3c3/dGame/dBehaviors/AreaOfEffectBehavior.cpp#L28

Reply:
Ah yeah I see that now.